db/migrate/20190626110700_add_columns_to_zipcode_rates.rb

```diff
@@ -0,0 +1,7 @@
+class AddColumnsToZipcodeRates < ActiveRecord::Migration
+  def change
+    add_column :zipcode_rates, :version, :integer
+
+    add_index :zipcode_rates, [:zip_code, :version] , using: :btree
```

Comment:
Can we swap the order of the columns in this index?  
```suggestion
    add_index :zipcode_rates, [:zip_code, :version] , using: :btree
    add_index :zipcode_rates, [:version, :zip_code] , using: :btree

```
I'm making an assumption that we'll only care about a `zip_code` for a given version.  That is, we're probably never going to look for records for a specific `zip_code` regardless of it's `version`, right?

Changing the order will improve efficiency when just filtering records on `version` too.

Reply:
That's correct.

Can do!